src/v2/Apps/Artwork/Components/ArtworkArtistSeries/ArtistSeriesArtworkRail.tsx

```diff
@@ -92,7 +92,6 @@ export const ArtistSeriesArtworkRail: React.FC<Props> = ({ artwork }) => {
               onClick={() =>
                 trackArtworkClick(artwork.slug, artwork.internalID, index)
               }
-              useLighterFont
```

Comment:
This prop goes away. Previously this was being used to switch between `black60` and `black100` on the sale message line only. It's being made consistently `black60` now.